packages/protocol/contracts/stability/interfaces/IMentoExchange.sol

```diff
@@ -6,21 +6,26 @@ pragma solidity ^0.5.13;
  */
 interface IMentoExchange {
   /**
-   * @notice Returns the output amount for a given input amount.
+   * @notice Returns the output amount and new bucket sizes for a given input amount.
```

Comment:
If updated buckets are needed, what would you think of having a wrapper (e.g. `getAmountOutBuckets`), that calls getAmountOut and calculates the new bucket sizes?